### ISSUE TITLE
algorithms app creator using nodeeditor

### ISSUE THREAD

**Issue by mohamadrezashafie** [paceholder/nodeeditor#287]
I need to algorithms app creator for benchmark test. so i have some component which can integrate them to create an app which work on inputs and create a file in output. so I have some question like below:
1) can I write a program which use some components and when i click on output button, implement a executable file which this exe app implement that algorithm (which I implement as input of node editor)? 
2) this exe can have ui? 
In other words, I need an app which its output is an exe app which this app use input components and have its UI?
and I can change algorithm (input component architecture) and get an another app with another implementation (in this example I need to use nodeeditor as a visualizer of algorith that user define it and output is exe which impelment that algorith..

**Comment by paceholder** [comment]
I need more input data to answer your question
